WLogger/WeekLogs/LogTableViewCell/LogViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LogViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/10/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+
+public struct LogViewModel {
```

Comment:
Why? Isn't it better since the model is immutable to avoid computing the variables each time they are needed?